Utilities/bootstrap

```diff
@@ -441,6 +441,14 @@ def build_swiftpm_with_cmake(args):
 
     build_with_cmake(args, cmake_flags, args.project_root, args.bootstrap_dir)
 
+    if platform.system() == 'Darwin':
```

Comment:
It's better to set the install name while building the PackageDescription libraries. You can do so by adding this in the Sources/PackageDescription/CMakeLists.txt file:

```
    if(CMAKE_SYSTEM_NAME STREQUAL Darwin)
      target_link_options(PD${PACKAGE_DESCRIPTION_VERSION} PRIVATE
        "SHELL:-Xlinker -install_name -Xlinker @rpath/PackageDescription.dylib")
    endif()
```

Reply:
Great! I initially trying playing with CMake's INSTALL_NAME_DIR, but it seems it only works when installing.